src/q5/algo-SINF1121/summary/algo-SINF1121-summary.tex

```diff
@@ -2177,6 +2177,7 @@ \subsubsection{Summary}
 
 
 \end{figure}
+\underlined{Note:} Kruskal's algorithm is generally slower than Prim's algorithm because it has to do a \texttt{connected()} operation for each edge, in addition to the priority-queue operations that both algorithms do for each edge processed.
```

Comment:
I don't know what is said in the book but in fact when Kruskal's algorithm is actually faster than Prim's algorithm.
The think is that it depends on how you implement the `connected()` operation. If you use an union find data structure then it is very fast and Kruskal will be faster since it just need to sort the edges at the beginning and Prim needs to maintain a heap when adding new elements in it.